knimin/lib/data_access.py

```diff
@@ -679,9 +685,11 @@ def format_survey_data(self, md):  # noqa
             md[1][barcode]['BODY_PRODUCT'] = md_lookup[site]['BODY_PRODUCT']
             md[1][barcode]['DESCRIPTION'] = md_lookup[site]['DESCRIPTION']
             md[1][barcode]['HOST_COMMON_NAME'] = md_lookup[site]['COMMON_NAME']
+
+            participant_name = dupes_lookup.get(
+                md[1][barcode]['SURVEY_ID'], specific_info['participant_name'])
             md[1][barcode]['HOST_SUBJECT_ID'] = sha512(
-                specific_info['ag_login_id'] +
-                specific_info['participant_name']).hexdigest()
+                specific_info['ag_login_id'] + participant_name).hexdigest()
```

Comment:
This shouldn't affect things though, as the case-sensitive names are also in the duplicates table.